File: existant/gestFormationWeb/gestion_formation_web/dto/DtoFormation.cs
using System;
using System.Collections.Generic;
using System.Linq;

using gestion_formation_web.dao;


namespace gestion_formation_web.dto
{
    public class DtoFormation 
    {
        public static formation Get(int idFormation)
        {
            return DaoFormation.Get(idFormation);
        }
        public static formation Get(string intitule)
        {
            return DaoFormation.Get(intitule);
        }
        public static IEnumerable<formation> Get()
        {
            return DaoFormation.Get();
        }

        public static void Add(formation uneformation)
        {
            DaoFormation.Add(uneformation);
        }

    }
}

File: Gestionnaire de formation/src/main/java/fr/esic/solutec/dto/DtoFormation.java
package fr.esic.solutec.dto;

import java.util.List;

import fr.esic.solutec.dao.Dao;
import fr.esic.solutec.dao.DaoFormation;
import fr.esic.solutec.entities.Formation;

public class DtoFormation {

	
	public static Formation getFormation(int idFormation) {
		
		return DaoFormation.getFormation(idFormation);
	}
	
	public static List<Formation> getListFormation() {
		
		return DaoFormation.getListFormation();
	}
	
	public static String[] getListNiveaux() {
		
		return Dao.getEnumValues("formation", "niveau");
	}
	
	
}
